exercises/mongo-shell/intermediate-read.md

```diff
@@ -162,3 +225,126 @@ db.movieDetails.find({"plot": {"$type": "string"}}).pretty()
 db.movieDetails.find({"plot": {"$type": "null"}}).pretty()
 ```
 ---
+
+## Array Operators
+
+In the following exercises, we'll look at operators for array fields. 
```

Comment:
It might be useful to make `array fields` at the end of this sentence a link to the Mongo docs for **Array Operators**.  This holds true for the other sections as well.  

Maybe once everything is down and can :ship: we will go back through and do this kind of thing.

Reply:
Refer to issue #10 !